tests/test_end_to_end.py

```diff
@@ -446,6 +453,11 @@ def test_end_to_end_search_download_cop_dataspace(self):
         expected_filename = "{}.zip".format(product.properties["title"])
         self.execute_download(product, expected_filename)
 
+    def test_end_to_end_search_download_cop_dataspace_s3(self):
+        product = self.execute_search(*COP_DATASPACE_S3_SEARCH_ARGS)
+        expected_filename = "{}.zip".format(product.properties["title"])
```

Comment:
```suggestion
        expected_filename = product.properties["title"]
```
not zip extension expected